component/file_cache/lru_policy.go

```diff
@@ -189,8 +183,6 @@ func (p *lruPolicy) CachePurge(name string) {
 	p.deleteEvent <- name
 }
 
-// Due to a race condition, this may return a false positive,
```

Comment:
Where is the fix for the race condition mentioned in the comments?

Reply:
The code that fixes this race condition is in `asyncCacheValid()`, where it now checks if a file is in the `nodeMap` before calling `cacheValidate()`.
The comment there explains the bug and the fix.